v23.1/cockroachdb-feature-availability.md

```diff
@@ -1,23 +1,53 @@
 
-## Session variables
+## Feature availability phases
 
-The table below lists the session settings that are available in preview. For a complete list of session variables, see [`SHOW {session variable}`](show-vars.html).
+Phase                                         | Definition | Accessibility 
+----------------------------------------------+------------+-------------
+Private preview                               | Feature is not production ready and will not be publicly documented. | Invite-only
+[Limited access](#features-in-limited-access) | Feature is production-ready but not available widely because of known limitations and/or because capabilities may change or be added based on feedback. | Feature is opt-in. To enroll your organization, contact your Cockroach Labs account team.
+[Preview](#features-in-preview)               | Feature is production-ready and publicly available. However, this feature may have known limitation and/or capabilities may change or be added based on feedback. | Public
+General availability (GA)                     | Feature is production-ready, publicly available, and has reached full development maturity. | Public
 
-| Variable                            | Default Value | Description                                                                                                                                                                                                                                                                                             |
-|-------------------------------------+---------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| `enable_experimental_alter_column_type_general`       | `'false'`       |  If set to `'true'`, enables [column type altering](#alter-column-types) for general cases, with some limitations.                                                                                                                                                                                   |
-| `experimental_enable_temp_tables`       | `'off'`       |  If set to `'on'`, enables [temporary objects](#temporary-objects), including [temporary tables](temporary-tables.html), [temporary views](views.html#temporary-views), and [temporary sequences](create-sequence.html#temporary-sequences).                                                                                                                                                                                   |
+## Features in limited access
```

Comment:
FYI I was initially tempted to suggest that we convert the subheadings in these to a table of features, however, I like that this format offers a place to document features that don't yet have a home. We may want to follow up to confirm that the current state of docs is OK for these. Regardless, it's better than having the info hidden exclusively in release notes!

Reply:
I can see that working too. The previous structure of this page was with these headings and blurbs so I went with that.... but now thinking about it, maybe it is better to have them in a table and just have the table link out to the relevant docs pages.

That being said, there are at least two features (show table fingerprints and SCRUB) that don't have docs, so these blurbs are all we have (and wouldn't fit nicely into a table)